roles/node/defaults/main.yml

```
@@ -35,6 +35,11 @@ node_force_restart: false
 # see https://github.com/ansible/ansible/issues/76855
 node_handler_id: ""
 
+### File used for Ansible annotation
+# File name for prometheus node-exporter textfile collector
+# Example:  /var/lib/prometheus/node-exporter/substrate.prom
+node_prometheus_file_exporter_path: ""
+node_prometheus_file_exporter_event: "node_playbook"
```

```suggestion
node_prometheus_file_exporter_event: "node_role"
```

@lazam,  maybe `node_role` is better name?